src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -17,10 +17,17 @@ namespace Auth0.AuthenticationApi
     public class AuthenticationApiClient : IAuthenticationApiClient, IDisposable
     {
         private readonly Uri _baseUri;
-        private readonly ApiConnection _apiConnection;
+        private IApiConnection _apiConnection;
```

Comment:
Does `ApiConnection` extend `IApiConnection`? Or am I missing a different relationship here?

Reply:
IApiConnection is the interface, ApiConnection is our concrete class.  By accepting an interface instead customers can create their own implementation that instead returns canned responses for their testing.